changelog.md

```diff
@@ -2,6 +2,13 @@
 
 //  Add your own contribution below
 
+* Add `danger.utils` DSL, which includes `danger.utils.href()` and `danger.utils.sentence()` - macklinu
+
+```js
+danger.utils.href("http://danger.systems", "Danger") // <a href="http://danger.systems">Danger</a>
+danger.utils.sentence(["A", "B", "C"]) // "A, B and C"
```

Comment:
Only a minor adjustment on _why_ it exists (I also added a "open to more ideas" too )

```
* Add `danger.utils` DSL, which includes `danger.utils.href()` and `danger.utils.sentence()` - macklinu

  We were finding that a lot of Dangerfiles needed similar functions, and so have added a `utils` object
  where we could offer functions that are going to be used across the board. If you can think of more 
  functions you use, we'd love to add them. Ideally you shouldn't need to use anything but Danger + utils
  to write your Dangerfiles.

  ```js
  danger.utils.href("http://danger.systems", "Danger") // <a href="http://danger.systems">Danger</a>
  danger.utils.sentence(["A", "B", "C"]) // "A, B and C"
  ``X

```